.github/workflows/deploy-snapshots.yml

```diff
@@ -3,7 +3,7 @@ name: Quarkus Deploy Snapshots
 on:
   schedule:
     - cron: '0 2 * * *'
-
+if: ${{ github.repository == 'quarkusio/quarkus' }}
```

Comment:
Are you sure it will work at this level? I don't see it documented at the top level here: https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions 

As far as I can see in the doc, it needs to be: `jobs.<job_id>.if`

Reply:
Shoot, wrong level! Thanks for pointing that out.

Reply:
Fixed.

PS: https://github.blog/changelog/2019-10-01-github-actions-new-workflow-syntax-features/